src/test/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessorTest.java

```diff
@@ -101,17 +103,24 @@ void GIVEN_one_system_component_one_file_less_than_max_WHEN_merge_THEN_reads_ent
         ignoreExceptionOfType(ec, DateTimeParseException.class);
         File file1 = new File(getClass().getResource("testlogs2.log").toURI());
         LogFile logFile1 = LogFile.of(file1);
-        String fileHash = logFile1.hashString();
+        SeekableByteChannel chan = Files.newByteChannel(logFile1.toPath(), StandardOpenOption.READ);
+        String fileHash = logFile1.hashString(chan);
         List<LogFileInformation> logFileInformationSet = new ArrayList<>();
         logFileInformationSet.add(LogFileInformation.builder().startPosition(0).logFile(logFile1).fileHash(fileHash).build());
+
         ComponentLogFileInformation componentLogFileInformation = ComponentLogFileInformation.builder()
                 .name("TestComponent")
                 .desiredLogLevel(Level.INFO)
                 .componentType(ComponentType.GreengrassSystemComponent)
                 .logFileInformationList(logFileInformationSet)
                 .build();
+
+        CloudWatchAttempt groupAttempt = new CloudWatchAttempt();
         logsProcessor = new CloudWatchAttemptLogsProcessor(mockDeviceConfiguration, defaultClock);
-        CloudWatchAttempt attempt = logsProcessor.processLogFiles(componentLogFileInformation);
+        groupAttempt.setLogGroupName(logsProcessor.getLogGroupName(componentLogFileInformation.getComponentType(),
+                "TestComponent"));
+        CloudWatchAttempt attempt = logsProcessor.processSingleLogFile(groupAttempt, componentLogFileInformation,
+                logFileInformationSet.get(0), chan);
         assertNotNull(attempt);
```

Comment:
We should remove this check given `processSingleLogFile` won't return the attempt a better assertion is to check if the contents got added to the attempt